engine/src/main/java/com/datatorrent/stram/stream/InlineStream.java

```diff
@@ -90,7 +91,7 @@ public void put(Object tuple)
   @Override
   public String toString()
   {
-    return "InlineStream{" + super.toString() + '}';
+    return getClass().getName() + '@' + Integer.toHexString(hashCode()) + "{reservoir=" + getReservoir().toString() + '}';
```

Comment:
@tweise verbose code or verbose output? The verbose output is used for logging and debugging.